telebot/types.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+from __future__ import annotations
```

Comment:
What's for?

Reply:
This is described in pep [563](https://peps.python.org/pep-0563/):
_This PEP proposes changing function annotations and variable annotations so that they are no longer evaluated at function definition time. Instead, they are preserved in __annotations__ in string form._

So, we can define class at any point of file and use it at any point; Basically, you don't have to define a class before using it - you can define it later, after using it.

Reply:
This was useful for using chatadministratorrights class before it's definition type, and i'm sure it is gonna be useful in future